rusty_pizza_server/src/order_model/order.rs

```diff
@@ -169,6 +169,21 @@ impl Order {
             })
         }
     }
+
+    pub fn user_has_to_pay(self, user: u32) -> Option<Money> {
+        match self.meals.get(&user) {
+            None => None,
+            Some(user_meals) => Some(user_meals.calculate_total_price()),
+        }
+    }
+
+    pub fn generate_pay_overview(self) -> HashMap<u32, Money> {
+        //TODO: replace userid with username from usermanagement after it is implemented (#29)
```

Comment:
Why do you want to change this? Imo it should be the responsibility of the caller to link the user IDs to the "user data" because we don't really need that here.

Reply:
Maybe we need both. One Method `HashMap<ID, Money>` and another `HashMap<Name, Money>`. This can help to reduce the calls between frontend and backend. 
How it is implemented at the end is a future discussion.
We coul let `generate_pay_overview` as it is and build an wrapper which replaces the IDs with the name, so the frontend can make one call to get the complete bill with names and prices.

Reply:
For now I will remove the TODO